src/navigation/index.cjsx

```diff
@@ -30,7 +30,6 @@ Navigation = React.createClass
     @forceUpdate() if @isMounted()
 
   close: ->
-    channel.emit('close.clicked')
     @context.close?()
 
```

Comment:
Nice catch.  Were we in essence firing close twice? Once by the `context.close()` and from whatever's listening to the `close.clicked` ?